sass/layouts/portfolio.scss

```diff
@@ -0,0 +1,77 @@
+.portfolio {
+  border-bottom: 6px solid $color-portfolio-border;
+  background: $color-portfolio-bg;
+  position: relative;
+  top: 66px;
```

Comment:
Why do you use such a style for top distance? Maybe padding would be more appropriate in such cases?

Reply:
Each section is pushed aside because at the top the "header" block displays: fixed;